tests/stability/cassandra_stress.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o pipefail
+
+# General env
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../metrics/lib/common.bash"
+
+IMAGE="docker.io/library/cassandra:latest"
+CONTAINER_NAME="${CONTAINER_NAME:-cassandra_test}"
+DOCKER_IMAGE="cassandra:latest"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
+CMD="cassandra -R"
+
+function main() {
+	local cmds=("docker")
+
+	init_env
+	check_cmds "${cmds[@]}"
+	sudo -E "${DOCKER_EXE}" pull "${DOCKER_IMAGE}"
+	sudo -E "${DOCKER_EXE}" save -o "${DOCKER_IMAGE}.tar" "${DOCKER_IMAGE}"
+	sudo -E "${CTR_EXE}" i import "${DOCKER_IMAGE}.tar"
+
+	sudo -E "${CTR_EXE}" run -d --runtime "${CTR_RUNTIME}" "${IMAGE}" "${CONTAINER_NAME}" sh -c "${PAYLOAD_ARGS}"
+	sudo -E "${CTR_EXE}" t exec --exec-id 1 "${CONTAINER_NAME}" sh -c "${CMD}"
```

Comment:
@GabyCT should these also be generated random ids rather than constant?

Reply:
@cmaf thanks changes applied